inst/qml/ContingencyTables.qml

```diff
@@ -45,14 +45,18 @@ Form
 
 		Group
 		{
-			CheckBox
+		    CheckBox
 			{
-				name: "oddsRatio"; label: qsTr("Log odds ratio (2x2 only)")
-				CIField { name: "oddsRatioConfidenceIntervalInterval"; label: qsTr("Confidence interval") }
+				name: "oddsRatioNormal"; label: qsTr("Odds ratio (2x2 only)")
+				CheckBox
+				{
+				    name: "LogOdds";     label: qsTr("Log Odds Ratio")
+				}
+				CIField { name: "oddsRatioConfidenceIntervalIntervalNormal"; label: qsTr("Confidence interval") }
 				RadioButtonGroup
 				{
 					title: qsTr("Alt. Hypothesis (Fisher's exact test)")
-					name: "oddsRatioHypothesis"
+					name: "oddsRatioHypothesisNormal"
```

Comment:
why did you change this? It leads to a lot of changes in the code, and I'm not sure what extra information the `"Normal"` adds here?